system/jlib/jsecrets.cpp

```diff
@@ -1066,6 +1066,25 @@ const MemoryAttr &getSecretUdpKey(bool required)
     return udpKey;
 }
 
+jlib_decl bool isPEMBuffer(const char *certificate)
```

Comment:
Might be worth coming up with a more intuitive function name (I didn't know what PEM stood for).  Something like containsEmbeddedKey()?

Reply:
ok, renamed function.